AxonDeepSeg/apply_model.py

```diff
@@ -74,7 +75,7 @@ def extract_from_nnunet_prediction(pred, pred_path, class_name, class_value) ->
     pred_path = ads_utils.convert_path(pred_path)
 
     if not np.any(pred == class_value):
-        raise ValueError(f'Class value {class_value} not found in the raw prediction.')
+        warnings.warn(f'Class value {class_value} not found in the raw prediction.')
```

Comment:
```suggestion
        logger.warning(f'Class value {class_value} not found in the raw prediction.')
```

Maybe we can use logger instead for prettier output in the console.